static/js/search.js

```diff
@@ -17,7 +17,7 @@
 
         for(var i = 1; i <= 10; i++){
             if(i > pages) break;
-            pageAnchors += '<a class="bing-page-anchor" href="/docs/search/?q='+searchTerm+'&page='+i+'">';
+            pageAnchors += '<a class="bing-page-anchor" href="search/?q='+searchTerm+'&page='+i+'">';
```

Comment:
How come this isn't:
```suggestion
            pageAnchors += '<a class="bing-page-anchor" href="/search/?q='+searchTerm+'&page='+i+'">';
```

Reply:
Yes, I will correct this. Thanks.
It was hard to find the numbers, at the bottom of the Bing search results page, which link to the next page results.